JCDispatch/src/main/java/com/openlattice/integrations/jcDispatch/Dispatch.java

```diff
@@ -0,0 +1,577 @@
+import retrofit2.Retrofit;
+
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+import static lib.nameParsing.*;
+
+public class Dispatch {
+
+    private static final Logger logger = LoggerFactory
+            .getLogger( Dispatch.class );
+
+    private static final Environment environment = Environment.LOCAL;
+
+    private static final DateTimeHelper dateHelper0 = new DateTimeHelper( DateTimeZone.forOffsetHours( -8 ), "yyyy-MM-dd HH:mm:ss.S" );
+
+    public static void main( String[] args ) throws InterruptedException {
+
+        FullQualifedNameJacksonDeserializer.registerWithMapper( ObjectMappers.getYamlMapper() );
+        FullQualifedNameJacksonDeserializer.registerWithMapper( ObjectMappers.getJsonMapper() );
+
+        final String personPath = args[ 0 ];
+        final String unitPath = args[ 1 ];
+        final String dispatchPath = args[ 2 ];
+        final String disTypePath = args[ 3 ];
+        final String jwtToken = args[ 4 ];
+
+        final SparkSession sparkSession = MissionControl.getSparkSession();
+
+        logger.info( "Using the following idToken: Bearer {}", jwtToken );
+
+        Retrofit retrofit = RetrofitFactory.newClient( environment, () -> jwtToken );
+        EdmApi edmApi = retrofit.create( EdmApi.class );
+        PermissionsApi permissionApi = retrofit.create( PermissionsApi.class );
+
+        Dataset<Row> person = sparkSession
+                .read()
+                .format( "com.databricks.spark.csv" )
+                .option( "header", "true" )
+                .load( personPath );
+
+        Dataset<Row> Unit = sparkSession
+                .read()
+                .format( "com.databricks.spark.csv" )
+                .option( "header", "true" )
+                .load( unitPath );
+
+        Dataset<Row> dispatch = sparkSession
+                .read()
+                .format( "com.databricks.spark.csv" )
+                .option( "header", "true" )
+                .load( dispatchPath );
+
+        Dataset<Row> disType = sparkSession
+                .read()
+                .format( "com.databricks.spark.csv" )
+                .option( "header", "true" )
+                .load( disTypePath );
+
+        RequiredEdmElements requiredEdmElements = ConfigurationService.StaticLoader
+                .loadConfiguration( RequiredEdmElements.class );
+
+        if ( requiredEdmElements != null ) {
+            RequiredEdmElementsManager manager = new RequiredEdmElementsManager( edmApi, permissionApi );
+            manager.ensureEdmElementsExist( requiredEdmElements );
+        }
+
+        logger.info( "ER Field names: {}", Arrays.asList( person.schema().fieldNames() ) );
+        logger.info( "ER Field names: {}", Arrays.asList( Unit.schema().fieldNames() ) );
+        logger.info( "ER Field names: {}", Arrays.asList( dispatch.schema().fieldNames() ) );
+        logger.info( "ER Field names: {}", Arrays.asList( disType.schema().fieldNames() ) );
+
+
+        Flight UnitMapping = Flight
```

Comment:
Java conventions are that variable names almost always start with a lowercase letter.